ext/RMagick/rmenum.c

```diff
@@ -797,12 +794,9 @@ FilterTypes_name(FilterTypes type)
 #if defined(IMAGEMAGICK_GREATER_THAN_EQUAL_6_8_9)
         ENUM_TO_NAME(LanczosRadiusFilter)
 #endif
-        // not a real filter name - defeat gcc warning message
```

Comment:
do we not need to handle the gcc warning anymore?

Reply:
No idea why that comment was there. Did not get a warning during the build and I also have no clue how you can get this warning.